test/scenarios/mappings/transformations.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "input": "$.products[*].id",
+    "output": "$.items[*].product_id"
+  },
+  {
+    "input": "$.products[*].name",
+    "output": "$.items[*].product_name"
+  },
+  {
+    "input": "$.products[*].category",
+    "output": "$.items[*].product_category"
+  },
+  {
+    "input": "$.products[*].(@.price * @.quantity * (1 - $.discount / 100))",
+    "output": "$.items[*].value"
+  },
+  {
```

Comment:
with this sum support, all our ecomm use-cases should be sufficing with this version